.github/workflows/test.yml

```diff
@@ -21,7 +21,12 @@ jobs:
         os: [ubuntu-latest]
         python-version: ["3.10"]
         test-subset:
-          - tests
+          - tests/model
+          - tests/statespace/test_statespace.py
+          - tests/statespace/test_kalman_filter.py
+          - tests/statespace --ignore tests/statespace/test_statespace.py --ignore tests/statespace/test_kalman_filter.py
+          - tests/distributions
+          - tests --ignore tests/model --ignore tests/statespace --ignore tests/distributions
       fail-fast: false
```

Comment:
Doesn't this make all run to completion? 

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#handling-failures

@ricardoV94

Reply:
I need a single job that I can add in the github UI as `required` for merging. You see the old ones that were in the system: 
![image](https://github.com/user-attachments/assets/a3f716b5-54b2-4a70-8f0d-7740eb2a5b29)

I don't want to have to type all the test names that now exist, and even that would be outdated when we change the config.

Reply:
This is how it looks in PyMC: https://github.com/pymc-devs/pymc/blob/5db3779b11a1e1d663447a3a78a6f1213b740409/.github/workflows/tests.yml#L382-L394

Reply:
done